extensions/amp-ooyala-player/0.1/amp-ooyala-player.js

```diff
@@ -103,31 +120,68 @@ class AmpOoyalaPlayer extends AMP.BaseElement {
 
   /** @override */
   pauseCallback() {
-    if (this.iframe_) {
+    // Only send pauseVideo command if the player is playing. Otherwise
+    // The player breaks if the user haven't played the video yet specially
+    // on mobile.
+    if (this.iframe_ && this.iframe_.contentWindow &&
+        this.playerState_ == PlayerStates.PLAYING) {
       this.pause();
     }
   }
 
+  /** @private */
+  handleOoyalaMessages_(event) {
+    if (event.origin != 'https://player.ooyala.com' ||
+        event.source != this.iframe_.contentWindow) {
+      return;
+    }
+    if (!event.data ||
```

Comment:
Yes, you are right, I removed it.   Now I need to fix "Error: timeout of 20000ms exceeded. Ensure the done() callback is being called in this test."  Any insights on it?